front/lib/api/agent_data_sources.ts

```diff
@@ -0,0 +1,57 @@
+import type { ConnectorProvider } from "@dust-tt/types";
+
+import type { Authenticator } from "@app/lib/auth";
+import { AgentDataSourceConfiguration } from "@app/lib/models/assistant/actions/data_sources";
+import { DataSource } from "@app/lib/models/data_source";
+import { User } from "@app/lib/models/user";
+import { Workspace } from "@app/lib/models/workspace";
+
+export type AgentEnabledDataSource = {
+  dataSourceId: number;
+};
+
+export async function getAgentEnabledDataSources({
```

Comment:
The name of this function is hard to parse as well.

Let's return what we need which is the count by and return a type that is more explicit in that direction 👍